pkgs/applications/networking/instant-messengers/wire-desktop/default.nix

```diff
@@ -129,6 +142,7 @@ let
     '';
   };
 
-in if stdenv.isDarwin
-  then darwin
-  else linux
+in
```

Comment:
This is another questionable formatting change. It's less clear that the conditional expression is in the body of the `let in`. What's the motivation for this?

I personally don't like the non-indented `then/else` clauses, this is not a statement but an expression, the extra indentation makes it much clearer the clauses are part of an expression.